packages/clay-color-picker/src/Custom.tsx

```diff
@@ -79,6 +79,7 @@ const ClayColorPickerCustom = ({
 
 					{showPalette && (
 						<button
+							aria-label={editorActive ? 'close' : 'custom color'}
```

Comment:
This was something out of scope but since there wasn't any description on this buton, I added an aria-label.
Is there any language key we can use on this?

Reply:
@ilzamcmed we don't have a language key because Clay is distributed outside of DXP too so we don't have `Liferay.Language.get` for example which is just processed in SSR instead we always expose an API to configure aria-labels, in that if we are adopting a pattern of exposing a `messages` API being the object with keys that are related to language, we can do the same here.